test/utils/wabt_engine.cpp

```diff
@@ -58,13 +58,19 @@ void WabtEngine::instantiate() {}
 
 bytes_view WabtEngine::get_memory() const
 {
+    if (m_env.GetMemoryCount() == 0)
+        return {};
```

Comment:
Can you add a comment to `wasm_engine.h` saying if empty memory is returned it means no memory is available (exported)?